aa_engine/aas_copyfromremote.m

```diff
@@ -19,11 +19,13 @@
     'allow404',     0, [0 1], ...     % 0 = crash on 404s? 1 = allow them?
     'allowcache',   aap.directory_conventions.allowremotecache, [-1 0 1], ...  % -1 = default to aap.directory_conventions.allowremotecache; 0 = force no; 1 = force yes
     'verbose',      logical(aap.options.verbose), [0 1], ...     % Display all those annoying "Retrieved..." messages?
+    'allowremotesymlinks', 0, [0 1] ...    % Allow symlinks when using remote connected pipelines; 1= allow 0= don't allow
```

Comment:
Why do you need this parameter? Why do not you simply rely on `aap.options.remotesymlinks`?